### DISCUSSION TITLE
The difference between standardized net benefit and net benefit

### DISCUSSION THREAD

**Discussion by leung1223** [MSKCC-Epi-Bio/decisioncurveanalysis#14] [Q&A] [answered]
Hello all! Recently, I've found that a few articles used standardized net benefits. In the calculation, Standardized Net Benefit = Net Benefit/Prevalence. However, I am not sure about the use of these two benefits. I would like to know the difference in interpretation and use of these two net benefits.  Thank you for your reply.

**Comment by VickersA** [comment, marked as answer]
The advantage of standardized net benefit is that you can compare net benefit between different studies that have different prevalence. The advantage of original (nonstandardized) net benefit, is that the unit is clinically interpretable and can allow extensions, for instance, taking into account test harm. 

The key point is that comparison between different studies is far less interesting and important than clinically interpretability and being able to conduct additional analyses such as incorporating test harm. That is why out of the many thousands of papers that have reported a decision curve, all but have a handful have reported traditional, nonstandardized net benefit.
